postprocessing/janus-pp-rec.c

```diff
@@ -671,8 +671,8 @@ int main(int argc, char *argv[])
 		if(data) {
 			/* Things are simpler for data, no reordering is needed: start by the data time */
 			gint64 when = 0;
-			bytes = fread(&when, sizeof(gint64), 1, file);
-			if(bytes < (int)sizeof(gint64)) {
+			int bytes = fread(&when, 1, sizeof(gint64), file);
```

Comment:
There's actually no need for the `int`, `bytes` is defined at line 341 already.

Reply:
Right.